Task 16.1.py

```diff
@@ -0,0 +1,52 @@
+class Person:
+    def __init__(self, *args, **kwargs):
+        self.name = args[0]
+        self.surname = args[1]
+        self.age = args[2]
+
+    def introduce(self):
+        return f"Hello, my name is {self.name} {self.surname}, I'm {self.age}"
+
+
+class Student(Person):
+    def __init__(self, name, surname, age, student_id, rating):

```

Comment:
`def __init__(self, student_id, rating, *args, **kwargs)` -> name, surname, age would go to *args